test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -141,32 +165,51 @@ def create_community_category(self, community_category_name, community_channel_n
         click_obj_by_name(CommunityScreenComponents.COMMUNITY_CREATE_CATEGORY_MENU_ITEM.value)
 
         wait_for_object_and_type(CreateOrEditCommunityCategoryPopup.COMMUNITY_CATEGORY_NAME_INPUT.value, community_category_name)
+        self._toggle_channels_in_category_popop(community_channel_names)
+        click_obj_by_name(CreateOrEditCommunityCategoryPopup.COMMUNITY_CATEGORY_BUTTON.value)
 
-        [loaded, listItem] = self._find_channel_in_category_popup(community_channel_name)
-        if loaded:
-            click_obj(listItem)
-        else:
-            verify_failure("Can't find channel " + community_channel_name)
+    def edit_community_category(self, community_category_name, new_community_category_name, community_channel_names):
+        [loaded, category] = self._find_category_in_chat(community_category_name)
+        verify(loaded, "Can't find category " + community_category_name)
```

Comment:
`verify` will show the message in all cases, positive and negative, so I suggest to add a more general message like: "Finding category : "